obelisk/python/obelisk_py/zoo/robot/hardware/leap_hand/leap_reset.py

```diff
@@ -0,0 +1,8 @@
+import dxl_motor_helper as dxl
```

Comment:
where does this get called? shouldn't this be in the `on_shutdown` callback of the `ObeliskLeapHand`?

Reply:
I originally used this as a test script, but now I added the functionality to the callback function.